### DISCUSSION TITLE
v1.3.1

### DISCUSSION THREAD

**Discussion by gabbybilka** [microsoft/WindowsAppSDK#3625] [Announcements]
## Windows App SDK 1.3.1(1.3.230502000) is now available

Windows App SDK 1.3.1 is a stable servicing release that fixes critical bugs for 1.3. 

- If you're building an app with the Windows App SDK for the first time, follow the [installation instructions here](https://docs.microsoft.com/windows/apps/winui/winui3/create-your-first-winui3-app?tabs=desktop-csharp).

- If you are already using the Windows App SDK 1.3 and would like to update to the latest release, you can do so by following the [update instructions here](https://docs.microsoft.com/windows/apps/windows-app-sdk/update-existing-projects-to-the-latest-release).

Check out our [release notes](https://learn.microsoft.com/windows/apps/windows-app-sdk/stable-channel#version-131-13230502000) for the list of issues fixed in this release.

### Updating to this version

The 1.3.1 package is available at https://www.nuget.org/packages/Microsoft.WindowsAppSDK/1.3.230502000

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WindowsAppSDK/releases/tag/v1.3.1'>v1.3.1</a>.</em>

**Comment by Xyncgas** [comment]
Question, did 1.2.230313.1 bug fix made it into this package

**Reply by Xyncgas** [reply]
@gabbybilka

**Reply by gabbybilka** [reply]
@bpulliam to confirm, but I believe 1.2.230313.1 (1.2.5) was included as part of 1.3.0 as well as 1.3.1.

**Reply by bpulliam** [reply]
Our process is to fix issues in main first, and then backport them so 1.3 should always have the fixes made in 1.2, even in servicing. You should not ever see missing bug fixes after taking an upgrade.
